### PR TITLE
Local IP adress are 127.0.0.1/8

### PR DESCRIPTION
**Pull request by St-Ex**
Personnally, I have multiple local apache (multi-project) and I use different local IP as server name.

This changes checks all local adress (from 127.0.0.1 to 127.255.255.255)
Localhost definition : https://fr.wikipedia.org/wiki/Localhost

It might be good to also add a message when not hasSW()
*I really spent a lot of time before finding this* :-)

### CODE REVIEW

**Comment by NekR** [comment]
Hi. Is it registering SW this way in both Firefox and Chrome?

**Comment by St-Ex** [comment]
Yep, just done the test.

File: tests/legacy/fixtures/basic-with-custom-time-update-service/__expected/main.js [hunk 1]
```diff
@@ -57,7 +57,7 @@
 	    // This is how I block Chrome 40 and detect Chrome 41, because first has
 	    // bugs with history.pustState and/or hashchange
 	    (window.fetch || 'imageRendering' in document.documentElement.style) &&
-	    (window.location.protocol === 'https:' || window.location.hostname === 'localhost' || window.location.hostname === '127.0.0.1')
+	    (window.location.protocol === 'https:' || window.location.hostname === 'localhost' || window.location.hostname.startsWith('127.'))
```
Comment (NekR) [inline tests/legacy/fixtures/basic-with-custom-time-update-service/__expected/main.js]: `indexOf('127.') === 0`

Comment (St-Ex) [inline tests/legacy/fixtures/basic-with-custom-time-update-service/__expected/main.js]: Done

**Comment by NekR** [comment]
Released: https://github.com/NekR/offline-plugin/releases/tag/v4.8.0

**Pull request merged**